features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapReceiverImpl.java

```diff
@@ -97,6 +107,19 @@ public void setTrapdConfig(TrapdConfiguration newTrapdConfig) {
         }
     }
 
+	@Override
+	public boolean equals(Object obj) {
```

Comment:
Also, equals() is defined on the wrong class, it should be defined on SnmpV3User instead.